provisioner/terraform/serve.go

```diff
@@ -31,14 +28,15 @@ type ServeOptions struct {
 	Tracer    trace.Tracer
 
 	// ExitTimeout defines how long we will wait for a running Terraform
-	// command to exit (cleanly) if the provision was stopped. This only
-	// happens when the command is still running after the provision
-	// stream is closed. If the provision is canceled via RPC, this
-	// timeout will not be used.
+	// command to exit (cleanly) if the provision was stopped. This
+	// happens when the provision is canceled via RPC and when the command is
+	// still running after the provision stream is closed.
 	//
 	// This is a no-op on Windows where the process can't be interrupted.
 	//
-	// Default value: 5 minutes.
+	// Default value: 3 minutes. This value should be kept less than the value
```

Comment:
Suggestion: Point to the actual default (`unhanger...`) to avoid docs becoming out of sync.